extensions/wikia/ImageReview/ImageCountGetter.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class ImageCountGetter extends WikiaModel {
+
+	const QUESTIONABLE = 'questionable';
+	const REJECTED     = 'rejected';
+	const UNREVIEWED   = 'unreviewed';
+	const INVALID      = 'invalid';
+
+
+	public function getImageCounts( int $reviewerId ) : array {
+		return $this->mergeCounts(
+			$this->getCountOfAllUnassignedImages(),
+			$this->getCountOfThisUsersImagesStillInReview( $reviewerId )
```

Comment:
Another change. We're now including outstanding reviews for the user in the total count. Beforehand, the counts would only include images which were waiting for review. This could be confusing in a case where the user was looking at images they have in review in "rejected" but with no more images to review afterwards. Even though they can see their current review (say, 10 images), the count would say 0. That' confusing. Now the user would see 10 (for the number of images they also have currently in review) which is more intuitive.